src/Controls/src/Core/Compatibility/Handlers/NavigationPage/iOS/NavigationRenderer.cs

```diff
@@ -1403,6 +1418,13 @@ protected override void Dispose(bool disposing)
 
 				if (disposing)
 				{
+					// Unsubscribe from toolbar item property changes
+					foreach (var item in _trackedToolbarItems)
```

Comment:
Isn t this code duplicated? Since we are calling `Disconnect(true)` that already has this code?

Reply:
Yep you are right!